src/Grid.elm

```diff
@@ -74,8 +74,8 @@ gridToCells grid =
 
 
 filter : (Cell -> Bool) -> Grid -> List Cell
-filter filter grid =
-    gridToCells grid |> List.filter filter
+filter filterVariable grid =
```

Comment:
this `filterVariable` could be called `predicate`. that's a commonly used named.

the official `List.filter` uses `isGood` tho lol

https://github.com/elm/core/blob/f88d6a6de98802f5cc2d99dc1a0c2734cc5bdd7b/src/List.elm#L213-L215